azure-local/update/update-best-practices.md

```diff
@@ -0,0 +1,110 @@
+
+1. **Use Azure Update Manager.**
+
+   AUM provides a centralized view to apply and manage updates across all Azure Local instances. In the Azure portal, go to **Azure Update Manager** > **Resources** > **Azure Local**.
+
+1. **Use filters to identify Azure Local resources ready for updates.**
+
+   Use filters like:
+   - Update Readiness = Healthy
+   - Status = updates available
+
+1. **Batch updates for large environments.**
+
+   Group clusters using:
+   - Tags
+   - Resource group
+   - Subscription
+   - Current version
+   - Location
+   
+   Define a model where you are updating in chunks.  
+
+1. **Test environment strategy.**
+
+   Select a few test clusters that mirror your Azure Loal resources in production. Run the update flow to validate before applying to production.
+
+1. **Production environment strategy.**
+
+   - Create batches using filter parameters.
+   - Start updating with the smallest batch during a maintenance window.
+   - If successful, proceed to the next batch.
+   - If issues arise:
+      - Investigate root cause
+      - Contact Microsoft support if needed
+      - Apply learnings to the future batches
+
+1. **Predownload update content to speed up update installation.**
+
+   You can predownload update content using the following update workflows. However, container images for Azure Resource Bridge (ARB) and Azure Kubernetes Service (AKS)  aren't included in the static payload. They are downloaded automatically during the update process.
+
+   - Prepare only workflow. See [Predownloads and check update readiness](./update-via-powershell-23h2.md#step-4-recommended-predownload-and-check-update-readiness).
+
+   - Offline update workflow. See [Import and discover update packages with limited connectivity](./import-discover-updates-offline-23h2.md).
+
+1. **Review known issues, logs, and use troubleshooting guides.**
+
+   - Before applying updates review the known issues for the target version and apply any recommended mitigations.
+   - If an update partially fails (for example, one node fails to update), Azure Update Manager will report the failure. Do not re-run the update blindly. Instead follow these steps:
+      - Retrieve error logs. Use the **Update history** view in the Azure portal to check error details for each node.
+      - Check offline logs. In disconnected scenarios, check event logs (for example, `ClusterAwareUpdating` log) on the affected node.
+      - Use troubleshooting guides. Microsoft publishes troubleshooting guides for Azure Local updates. For example, steps to retry an update that stuck at a certain phase. Follow those guides to reset any update run state or clear maintenance mode if needed. Ensure the cluster is stable before retrying the update.
+
+## Don’ts for Azure Local updates
+
+Avoid the following practices when managing Azure Local updates, as they can lead to errors, downtime, or unsupported states.
+
+1. **Don't update Azure Local machines individually.**
```

Comment:
we need to make it more generic and not even mention any other tools. I suggest we make this guidance more generic to not update Azure Local with tools other than AUM.